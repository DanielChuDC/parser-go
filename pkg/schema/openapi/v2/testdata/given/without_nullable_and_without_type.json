{
  "payload": {
    "example": {
      "test": "example1"
    },
    "examples": [
      {
        "test": "example0"
      }
    ],
    "properties": {
      "test": {
        "description": "hello"
      }
    },
    "additionalProperties": {
      "description": "test"
    }
  }
}