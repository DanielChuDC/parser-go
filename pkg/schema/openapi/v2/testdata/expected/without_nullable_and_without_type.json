{
  "payload": {
    "additionalProperties": {
      "description": "test"
    },
    "properties": {
      "test": {
        "description": "hello"
      }
    },
    "examples": [
      {
        "test": "example0"
      },
      {
        "test": "example1"
      }
    ]
  }
}